fdmi/plugins/fdmi_app

```diff
@@ -118,13 +216,21 @@ def process_fdmi_record(kv_record, p):
     # Check de-dup records
     kv_i = kv_records.get(keystr)
```

Comment:
Is this keystr good to eliminate duplicates? What if we for example add a tag only (bucket+name+version will be same I assume), will it keep it or ignore it?

Reply:
I have discussed this with Rajkumar and Priyanka.

This keystr is just an example how to do de-duplication.
For real case, we need to find out what is the best `keystr` to do de-duplication.
For example, "Bucket-name/Object-name" might be suitable, or with some additional fields.

Reply:
+1